README.md

```diff
@@ -113,6 +113,7 @@ In addition, integration modules are available for the following libraries, prov
 
 ### External projects using Optuna
 
+* [Hugging Face Trainer's Hyperparameter Search](https://huggingface.co/docs/transformers/main/main_classes/trainer#transformers.Trainer.hyperparameter_search)
```

Comment:
Not a strong opinion, but how about stop using the title case like other entries?
```suggestion
* [Hugging Face Trainer's hyperparameter search](https://huggingface.co/docs/transformers/main/main_classes/trainer#transformers.Trainer.hyperparameter_search)
```

Reply:
Thanks, that makes sense.